### PR TITLE
Test against Postgres 11

### PR DESCRIPTION
**Pull request by Majkl578**
|      Q       |   A
|------------- | -----------
| Type         | improvement

#### Summary

PostgreSQL 11 beta 1 [will be released tomorrow](https://www.postgresql.org/message-id/12B4300D-2171-4690-A020-E9852797F478%40excoventures.com). It's already tagged and available through Debian experimental, so let's use it on CI for preliminary testing and spotting possible breakages.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -319,6 +320,31 @@ jobs:
...
 
+    - stage: Test
+      php: 7.1
+      env: DB=pgsql POSTGRESQL_VERSION=11.0
+      sudo: required
+      services:
+        - docker
+      before_script:
+        - bash ./tests/travis/install-postgres-11.sh
+    - stage: Test
```
Comment (morozov) [inline .travis.yml]: Could you visually separate jobs by adding an empty line between them?

**Review by morozov — DISMISSED** [review]
Not interesting, all tests passed :smile:

File: .travis.yml (hunk 1 above)
Comment (Majkl578) [reply .travis.yml]: Stages are currently separated by vendor&version so this is consistent with the rest.

**Comment by Majkl578** [comment]
Hmm, actually there is an issue with some existing postgres on Travis, even though I removed `service: - postgresql`. :thinking:

**Comment by Majkl578** [comment]
Looks ok now.
```
+sudo service postgresql stop
 * Stopping PostgreSQL 9.2 database server        * Stopping PostgreSQL 9.3 database server        * Stopping PostgreSQL 9.4 database server        * Stopping PostgreSQL 9.5 database server        * Stopping PostgreSQL 9.6 database server
```

**Review by morozov — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]

**Pull request merged**